### PR TITLE
[WIP] Add Farsi translation

### PR DESCRIPTION
**Pull request by huey735**
For some reason "How does the Bisq DAO work?" under bisq.network/fa/faq won't expand to show the text. @wiz @m52go @erciccione

### CODE REVIEW

**Comment by wiz** [comment]
Can you deploy this to netlify for testing? Sign up using GitHub at https://app.netlify.com/signup and add your fork and it will deploy to something.netlify.com

**Comment by wiz** [comment]
Actually @m52go can you enable "preview builds" for PRs like this on Netlify so @huey735 doesn't have to do that? The bisq-docs repo already does this, is very convenient. Thanks

**Comment by m52go** [comment]
Sure I'll reach out to cbeams to see why it stopped.

**Comment by huey735** [comment]
@wiz, Yeah I tried Netlify for the first time with the default settings and got an error when deploying.

File: _data/fa.yml [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+  hBisqDAO: The Bisq DAO
+  hBisqDaoWhat: "BISQ DAO چیست؟"
+  pBisqDaoWhat: >
+    <p>BISQ DAO (سازمان خودمختار غیر متمرکز) یک مکانیزم اداری غیرمتمرکز برای نرم افزار Bisq است. که برای یک شرکت است. اگر یک نرم افزار غیر متمرکز توسط یک نهاد واحد مثل یک شرکت کنترل شود, چقدر می تواند مفید باشد؟</p> <p>یک نرم افزار غیر متمرکز بدون بدون حکمرانی غیر متمرکز مفید واقع نخواهد بود.</p> <p>به همین دلیل است که Bisq یک شرکت یا نهاد قانونی از هر نوعی نیست - بلکه به عنوان یک DAO سازماندهی شده است. DAO برنامه های مالی و استراتژی های نرم افزاری را مدیریت می کند تا شبکه Bisq را تقویت کند و در عین حال آن را در برابر حملات به زیرساخت هایی که رهبری و عملیات آن را کنترل می کند، سخت می کند.</p> <p>به طور خلاصه، Bisq DAO قادر است Bisq را <a href='https://docs.bisq.network/intro.html' target='_blank'>از همان ابتدا به یک اصل اصلی پروژه</a> تبدیل کند که بیشتر مقاوم در برابر سانسور باشد.</p> <p>در این مقدمه، بیشتر درباره Bisq DAO بخوانید.<a href='https://docs.bisq.network/user-dao-intro.html' target='_blank'>.</p>
```
Comment (erciccione) [inline _data/fa.yml]: You open a hyperlink at the end of this string, but you don't close it. This is messing up the format of the section that follows. I removed the link locally while testing and everything worked fine.

**Review by erciccione — CHANGES_REQUESTED** [review]
Found the problem. See comment

File: _data/fa.yml (hunk 1 above)
Comment (huey735) [reply _data/fa.yml]: Thanks, @erciccione. After some searching I've found it yesterday. @Blockchainsmoker has taken care of it and I'll update it soon.

**Comment by erciccione** [comment]
@huey735 Could you rebase and squash those 5 commits?

**Pull request closed without merging**